api/go.mod

```diff
@@ -6,7 +6,8 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.2
 	github.com/google/go-cmp v0.5.7
-	github.com/gravitational/trace v1.1.17
+	github.com/gravitational/teleport v0.0.0
```

Comment:
Yea, we can't do this. You'll have to move whatever you depend on into the API module to break this dependency.

Reply:
What do you think about I create a `github.com/gravitational/teleport/contants` ?
I need to fetch the TraitXYZ from the `teleport/contants.go`

Reply:
Mentioned in another thread, but I would just move the Trait constants to api/.

Reply:
I agree that `constants` is terrible, but I'm not sure `api` or `types` are that much better in this case, so I'll leave it up to you. My 2c is that the constants should be part of the package that owns them, not pushed to some generic placeholder of constants.

Reply:
Agree :+1: 
Thank you both :bow: